### DISCUSSION TITLE
Icons resizes on zoom

### DISCUSSION THREAD

**Discussion by ventoline** [visgl/deck.gl#8164] [Q&A] [answered]
the icons scales up on zoom out, and vice versa,  how can i keep the size proportional (as for the scatterplot layer)? Or better, load a custom texture for a scatterplot layer?

**Comment by Pessimistress** [comment, marked as answer]
[sizeUnits](https://deck.gl/docs/api-reference/layers/icon-layer#sizeunits)

**Comment by ventoline** [comment]
great thanks!
